07-Linking-Accounts/Auth0Sample/ProfileViewController.m

```diff
@@ -137,7 +137,7 @@ - (IBAction)linkAccount:(id)sender {
                 [self showErrorAlertWithMessage:error.localizedDescription];
             });
         } else {
-            [auth linkUserAccountWithIdToken:[keychain stringForKey:@"id_token"] userId:self.userProfile.sub otherAccountToken:credentials.idToken callback:^(NSError * _Nullable error, NSArray<NSDictionary<NSString *,id> *> * _Nullable payload) {
```

Comment:
Does this actually work?  I'm not an objective C or Swift person but it looks to me like it doesn't work even before the PR.

Surely this should just be `[auth linkUserAccount:[keychain...` as the "withAccessToken is just the initial parameter name and doesn't form part of the method name in either Objective-C or Swift.

Reply:
So this is a little magical as hybrid. The swift func is

```swift
@objc
    func unlinkUserAccount(withAccessToken accessToken: String, userId: String, identity: Identity, callback: @escaping (Error?, [[String: Any]]?) -> ())
```

With the @objc attribute for access in ObjC project.

This will be magically transformed into

```objc
[auth unlinkUserAccountWithAccessToken: ...
```

Reply:
I ran it locally to be sure